modules/nf-core/vsearch/fastqfilter/main.nf

```diff
@@ -0,0 +1,51 @@
+
+process VSEARCH_FASTQFILTER {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/vsearch:2.28.1--h6a68c12_1':
+        'biocontainers/vsearch:2.28.1--h6a68c12_1' }"
+
+    input:
+    tuple val(meta), path(fastq)
+
+    output:
+    tuple val(meta), path('*.filtered.fasta')   , emit: fasta
+    path "*.filtered.log"            , emit: log
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    vsearch \\
+        --fastq_filter ${fastq} \\
+        $args \\
+        --fastaout ${prefix}.filtered.fasta 2>&1 | tee ${prefix}.filtered.log
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        vsearch: \$(vsearch --version 2>&1 | head -n 1 | sed 's/vsearch //g' | sed 's/,.*//g' | sed 's/^v//' | sed 's/_.*//')
```

Comment:
This seems unlikely to be minimal. What does the string look like out of vsearch?

Reply:
Trying the command gives me `2.28.1`, which seems okay. Also, the `version.yml` from nf-test looks like this:

```
"VSEARCH_FASTQFILTER":
    vsearch: 2.28.1
```

Reply:
Yes, but you could probably get the string out without 5 separate commands is what I meant.

Reply:
Oh, sorry. Yes, you are right.